lib/display.rb

```diff
@@ -0,0 +1,33 @@
+class Display
+
+  def initialize(console = Console.new, message)
+    @console = console
+    @message = message
+  end
+
+  public
+
+  def present_board(grid)
+    position_in_array = 0
+    while position_in_array < grid.size
+      @console.present(grid.slice(position_in_array, size_of_grid(grid)).join(" "))
+      position_in_array += size_of_grid(grid)
+    end
+  end
+  
+  def size_of_grid(grid)
```

Comment:
You have a method `dimension()` in board that is exactly the same as this... I wonder how you could remove this duplication? Perhaps passing the dimension around rather than the size?

Also the naming here suggests that the `size_of_grid` is the `dimension`, but in the `board` class the two things are different.